plugins/superio/fu-superio-device.c

```diff
@@ -448,6 +448,8 @@ fu_superio_device_init (FuSuperioDevice *self)
 {
 	fu_device_set_physical_id (FU_DEVICE (self), "/dev/port");
 	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_INTERNAL);
+	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_CAN_VERIFY);
+	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_CAN_VERIFY_IMAGE);
```

Comment:
can't we modify `fu_device_add_flag` so that adding FWUPD_DEVICE_FLAG_CAN_VERIFY_IMAGE automatically adds FWUPD_DEVICE_FLAG_CAN_VERIFY?

Reply:
Fixed